crates/astria-sequencer/src/app/mod.rs

```diff
@@ -220,6 +222,15 @@ impl App {
         state_tx.put_base_prefix(&genesis_state.address_prefixes().base);
 
         crate::asset::initialize_native_asset(genesis_state.native_asset_base_denomination());
+        let native_asset = crate::asset::get_native_asset();
+        if let Some(trace_native_asset) = native_asset.as_trace_prefixed() {
+            state_tx
+                .put_ibc_asset(trace_native_asset)
+                .context("failed to put native asset")?;
+        } else {
+            bail!("native asset must not be in ibc/<ID> form")
+        }
```

Comment:
what's going on with indent here? it feels like there is a missing block here?

Reply:
The indent is actually fine - github code wrapping makes this look a bit weird if your window is too narrow.

Reply:
Oh wow that was cursed. dammit github